docs/Project-Board-Guide.md

```diff
@@ -0,0 +1,22 @@
+# Simorgh Project Board Guide
+
+Overview of a Github Project: https://help.github.com/en/articles/about-project-boards
+
+BBC's Simorgh Project board: https://github.com/orgs/bbc/projects/20
+
+### Overiew of each column
+
+<!-- prettier-ignore -->
+| Column Name               | Issues/PRs that belong in this column         | When to move card to the next column | Additional context |
+| ------------------------- | ----------------------------------------------| ------------------------------------ |------------------- |
+| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined to the corresponding team that has been labelled to the ticket, then it can be manually moved to the "Ready for Dev". | n/a |
+| Ready for Dev | Contains issues that have been refined | Once you have selected an issue to work on, you must assign yourself to it and manually move the issue to the "Issues in progress".| n/a |
+| Externally blocked | Contains issues/PRs that are blocked my an external factor | Manually move the issue/PR when it is no longer blocked | n/a |
+| Issues in progress | Contains issues that are in working progress | Cards in this column will move automatically to "Done", once the PR corresponding the the issue has been merged and closed. | n/a |
```

Comment:
They only move to done if https://help.github.com/en/articles/closing-issues-using-keywords are used in the PR. 

Also add note of caution at over using keywords. PRs should not use keywords unless they definitely complete the entire issue. There are often many PRs per issue